config/localstack/1-setup-sqs.sh

```diff
@@ -13,3 +13,6 @@ done
 
 aws sqs --endpoint-url=http://localstack:4566 create-queue --queue-name case-creator-queue-dlq
 aws sqs --endpoint-url=http://localstack:4566 create-queue --queue-name case-creator-queue --attributes '{"RedrivePolicy": "{\"deadLetterTargetArn\":\"arn:aws:sqs:eu-west-2:000000000000:case-creator-queue-dlq\",\"maxReceiveCount\":1}"}'
+
+aws sqs --endpoint-url=http://localstack:4566 create-queue --queue-name migration-queue-dlq
```

Comment:
We shouldn't need these queues as you are specifying the config within deployment time.

Reply:
Wouldn't it be useful during the development and testing locally?

Reply:
You can just use the case-creator-queue. 

Effectively case creator is going to listen to the UKVI queue or your queue in real life but the underlying behaviour is much the same. It will still go through the same Listener and the correct message handler will be picked off the message type. 

You can therefore use the top queue and pass through a message 'type' of `MIGRATION` and it should work (if set up correctly).

Reply:
Fixed